python/tvm/driver/tvmc/model.py

```diff
@@ -280,7 +284,10 @@ def export_package(
                 executor_factory, package_path, cross, cross_options, output_format
             )
         elif output_format == "mlf":
-            package_path = export_model_library_format(executor_factory, package_path)
+            if export_model_library_format is not None:
+                package_path = export_model_library_format(executor_factory, package_path)
+            else:
+                raise Exception("micro tvm cannot be loaded, verify USE_MICRO in config.cmake")
```

Comment:
I think it doesn't hurt to be more explicit on what we think is wrong, so I'd like to use the same message as in `session.py`, i.e.:

```
ImportError: micro tvm is not enabled. Set USE_MICRO to ON in config.cmake
```

Reply:
I would like to re-throw an original exception, but would not like to complicate and store one more attribute in global scope. Ok, let's to copy-past message

Reply:
fixed

Reply:
@elvin-n Yeah, I meant only copy-past the message, not the exception type.